config/schemas/config.yaml

```diff
@@ -65,40 +71,110 @@ $defs:
         - boolean
     properties:
       chunkLimitSize:
+        title: Chunk limit size for Fluentd buffer
+        description: |
+          Events will be written into chunks until the size of chunks become `chunkLimitSize`
+
+          [Buffering parameters](https://docs.fluentd.org/configuration/buffer-section#buffering-parameters)
         examples:
           - 50MB
         type: string
       flushInterval:
+        title: Flush interval for Fluentd buffer
+        description: |
+          Flushes the buffer each `flushInterval`, if [flushMode](#flushmode) is equal to `interval`
+
+          [Flushing parameters](https://docs.fluentd.org/configuration/buffer-section#flushing-parameters)
         examples:
           - 15m
         type: string
       flushMode:
-        examples:
+        title: Flush mode for Fluentd buffer
+        description: |
+          [Flushing parameters](https://docs.fluentd.org/configuration/buffer-section#flushing-parameters)
+        enum:
+          - lazy
           - interval
+          - immediate
+        meta:enum:
+          lazy: "Flushes/writes chunks once per timekey"
+          interval: "Flushes/writes chunks per specified time via flushInterval"
```

Comment:
```suggestion
          interval: "Flushes/writes chunks per specified time via `flushInterval`"
```

Reply:
Styling inside the enum table does not work :cry: